### PR TITLE
Added drop down logic to pick and choose which method to use

### PR DESCRIPTION
**Pull request by jpamarthi**
1. Bumping the version to 0.41
2. Used new color palettes for a better look. This is still evolving. 
3. Added drop down logic to pick and choose different methodologies
4. Removed results.html and moved everything to index.html to simplify things.

image

image

image

image

### CODE REVIEW

File: src/pycatcher/catch.py [hunk 1]
```diff
@@ -651,5 +651,6 @@ def detect_outliers_stl(df) -> Union[pd.DataFrame, str]:
             print(return_outliers)
 
         logging.info("Completing outlier detection using STL")
+        return return_outliers
```
Comment (jpamarthi) [inline src/pycatcher/catch.py:654]: Return statement is missing for some reason!

File: pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pycatcher"
-version = "0.0.40"
+version = "0.0.41"
```
Comment (jpamarthi) [inline pyproject.toml:3]: Bumped the version

File: src/pycatcher/catch.py (hunk 1 above)
Comment (jpamarthi) [reply src/pycatcher/catch.py:654]: @aseemanand could you please check and confirm if I got this right?

Comment (aseemanand) [reply src/pycatcher/catch.py:650]: We can remove print if return is used. It’ll re-print outliers if both are used. I can make the change.

**Comment by jpamarthi** [comment]
I started with these 3 options. Please let me know if you have any suggestions.
image

**Review by aseemanand — APPROVED** [review]
LGTM

**Pull request merged**